client/handler.go

```diff
@@ -307,6 +441,64 @@ func (h *Handler) Get(w http.ResponseWriter, r *http.Request, ps httprouter.Para
 	h.r.Writer().Write(w, r, c)
 }
 
+// swagger:route GET /dyn-clients/{id} public getDynOAuth2Client
+//
+// Get an OAuth 2.0 Client.
+//
+// Get an OAUth 2.0 client by its ID. This endpoint never returns passwords.
+//
+// OAuth 2.0 clients are used to perform OAuth 2.0 and OpenID Connect flows. Usually, OAuth 2.0 clients are generated for applications which want to consume your OAuth 2.0 or OpenID Connect capabilities. To manage ORY Hydra, you will need an OAuth 2.0 Client as well. Make sure that this endpoint is well protected and only callable by first-party components.
+//
+//
+//     Consumes:
+//     - application/json
+//
+//     Produces:
+//     - application/json
+//
+//     Schemes: http, https
+//
+//     Responses:
+//       200: oAuth2Client
+//       401: genericError
+//       500: genericError
+func (h *Handler) GetDynamicRegistration(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	var id = ps.ByName("id")
+	var secret = r.URL.Query()["secret"][0]
```

Comment:
Having passwords or secrets in the URL query is bad security practice as it will be exposed in e.g. HTTP logs. Please use Basic Authorization instead